go/client/cmd_team_accept_invite.go

```diff
@@ -30,6 +31,10 @@ func newCmdTeamAcceptInvite(cl *libcmdline.CommandLine, g *libkb.GlobalContext)
 				Name:  "token",
 				Usage: "token",
 			},
+			cli.BoolFlag{
+				Name:  "seitan",
```

Comment:
We should never let the user see this term.

Reply:
Can we just tell the difference between the two types of tokens based on length?  That's the easiest thing I can think of.